src/components/MeetYourhostSection/MeetYourHostSection.jsx

```diff
@@ -0,0 +1,32 @@
+import MeetYourHost from "../MeetYourHostRight/MeetYourHostRight";
```

Comment:
I believe that it is better to put "MeetYourHostRight" and "MeetYourHostLeft" folders inside the "MeetYourHostSection" folder.